dd-java-agent/agent-iast/src/main/java/com/datadog/iast/taint/Ranges.java

```diff
@@ -217,6 +243,36 @@ public static Range[] newArray(final long size) {
     return new Range[size > MAX_RANGE_COUNT ? MAX_RANGE_COUNT : (int) size];
   }
 
+  /** Insert the new range maintaining order (it assumes that both arrays are already sorted) */
+  public static Range[] insert(final Range[] currentRanges, final Range... newRanges) {
```

Comment:
```suggestion
  public static Range[] append(final Range[] currentRanges, final Range... newRanges) {
```

Can we call this `append` or `concat`? `insert` sounds like an in-place operation.

Reply:
Changed into `mergeRangesSorted` as it makes more sense as in a merge sort algo (also rename the vars to make it clear what it does)